File: listings/if_example.m
% if ( CONDITION ) STATEMENT
% [elseif STATEMENT ]
% [else STATEMENT ]
% end.
% 
% EXAMPLE: What are we gonna 
% do today?
%

day=weekday(now);

if (day == 6 )
   disp('PUB!')
elseif (day == 1 || day == 7)
      disp('sleep')
else
   disp('duh.') 
end

File: listings/if_example.py
# if CONDITION: 
#    STATEMENT
# [elif CONDITION: 
#    STATEMENT ]
# [elif...]
# [else:
#    STATEMENT ]
#
import datetime

#Get day of the week as an integer, 
#Monday is 0, Sunday is 6.
day = datetime.datetime.today().weekday()

if day == 2:
   print('New PAG Package')
elif day == 5 or day == 6:
   print('weekend')
else:
   print('Deal with other stuff.') 

